lib/systems/platforms.nix

```diff
@@ -255,6 +255,7 @@ rec {
       arch = "armv6";
       fpu = "vfp";
       float = "hard";
+      abi = "aapcs-linux";
```

Comment:
I think it's better to drop this and make the `release-cross.nix` one match the one in `make-bootstrap-tools.nix` (unless it somehow breaks builds).